.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -34,6 +30,6 @@ _Explain how this change can be tested (or why it can't be tested)_
 - [ ] Comparison screenshots added for visual changes
 - [ ] Affected documentation updated (docusaurus, tutorial, CMS)
 
-### 🎉 GIF
+### 🤡 Meme
```

Comment:
Could we replace the 🤡 with 🎁 or 🌠 or similar image emoji?
The 🤡 emoji carries a bad connotation with it - mostly on the social media and our daily communications via chat :/
Thanks.

Reply:
Yes, make sense! Nice spot, will change this!

Reply:
Done! ✅